doc/source/user_guide/io.rst

```diff
@@ -1907,6 +1908,18 @@ Note ``NaN``'s, ``NaT``'s and ``None`` will be converted to ``null`` and ``datet
    json = dfj.to_json()
    json
 
+.. note::
+
+   When using ``orient='table'`` along with user-defined ``ExtensionArray``,
+   the generated schema will contain an additional ``extDtype`` key in the respective
+   ``fields`` element. This extra key is not standard but does enable JSON roundtrips
+   for extension types (e.g. ``read_json(df.to_json(orient="table"), orient="table")``).
+
+   The ``extDtype`` key carries the name of the extension, if you have properly registered
+   the ``ExtensionDtype``, pandas will use said name to perform a lookup into the registry
+   and re-convert the serialized data into your custom dtype.
```

Comment:
Do you mean something like `json.loads`?

Reply:
No, what would be good to do is to validate the JSON output from `to_json()` against the JSON schema here: `https://specs.frictionlessdata.io/schemas/table-schema.json` .  If you use the `jsonschema` package, and write a small python program that just validates your output against that schema, and post the results here, that would suffice. (In other words, don't add `jsonschema` to the tests in the pandas source)

Reply:
this is not appropriate here, it should be with the 'table' docs.

Reply:
@Dr-Irv here is the answer to your question 
```
~
[16:13:25] Φ jsonschema --instance test.json Downloads/table-schema.json
{'schema': {'fields': [{'name': 'index', 'type': 'integer'}, {'name': 'd', 'type': 'any', 'extDtype': 'DateDtype'}], 'primaryKey': ['index'], 'pandas_version': '1.4.0'}, 'data': [{'index': 0, 'd': '2021-12-08T00:00:00.000Z'}]}: 'fields' is a required property
~
[16:13:51] Φ vim test_int.json
~
[16:15:49] Φ jsonschema --instance test_int.json Downloads/table-schema.json
{'schema': {'fields': [{'name': 'index', 'type': 'integer'}, {'name': 'd', 'type': 'integer'}], 'primaryKey': ['index'], 'pandas_version': '1.4.0'}, 'data': [{'index': 0, 'd': 1}, {'index': 1, 'd': 2}, {'index': 2, 'd': 3}]}: 'fields' is a required property
~
```

Reply:
@jmg-duarte It failed, because you tested the entire JSON that includes data, which is not part of the schema.  You'd have to only validate the JSON in the `schema` part, e.g.
```
{
  "fields": [
    { "name": "index", "type": "integer" },
    { "name": "d", "type": "any", "extDtype": "DateDtype" }
  ],
  "primaryKey": ["index"],
  "pandas_version": "1.4.0"
}
```

Reply:
Oh I see, thank you! Here's the fixed version then:
```
[17:25:22] Φ bat test.json
───────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: test.json
───────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ {"fields":[{"name":"index","type":"integer"},{"name":"d","type":"any","extDtype":"DateDtype"}],"primaryKey":["index"],"pandas_version"
       │ :"1.4.0"}
───────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
~
[17:25:30] Φ jsonschema --instance test.json Downloads/table-schema.json
~
```

Reply:
> Seems to me that it is working!

Yes, I agree.

Reply:
Seems to me that the only thing left to do is for @jreback to get back to me on my replies so I can finish, ensure it build and ship (?)

Reply:
@jmg-duarte Based on the comment from @jreback https://github.com/pandas-dev/pandas/pull/44722/files#r762492051 can you move this text:
```
.. note::

   When using ``orient='table'`` along with user-defined ``ExtensionArray``,
   the generated schema will contain an additional ``extDtype`` key in the respective
   ``fields`` element. This extra key is not standard but does enable JSON roundtrips
   for extension types (e.g. ``read_json(df.to_json(orient="table"), orient="table")``).

   The ``extDtype`` key carries the name of the extension, if you have properly registered
   the ``ExtensionDtype``, pandas will use said name to perform a lookup into the registry
   and re-convert the serialized data into your custom dtype.
```
to after this line related to the table schema:
```
.. _Table Schema: https://specs.frictionlessdata.io/table-schema/
```